lib/protocols/password_encoder.ex

```diff
@@ -0,0 +1,18 @@
+defprotocol EaRestaurantDataLoader.PasswordEncoder do
+    def validate_password(pass_data)
+    def encode_password(pass_data)
+end
+
+alias EaRestaurantDataLoader.Base64
+
+defimpl EaRestaurantDataLoader.PasswordEncoder, for: Base64 do
```

Comment:
can you move this to the move this to its own file?

Same comments about the namespace

Reply:
Done.